src/main/kotlin/org/malachite/estella/people/api/UserController.kt

```diff
@@ -65,33 +66,29 @@ class UserController(
     fun refresh(
         @PathVariable userId: Int,
         @RequestHeader(EStellaHeaders.jwtToken) jwt: String?
-    ): ResponseEntity<Message> {
+    ): ResponseEntity<Any> {
         jwt ?: return OwnResponses.UNAUTH
         return securityService.refreshToken(jwt, userId)
-            ?.let { ResponseEntity.ok()
+            ?.let {
+                ResponseEntity.ok()
                     .header(EStellaHeaders.authToken, it)
                     .header(HttpHeaders.ACCESS_CONTROL_EXPOSE_HEADERS, loginExposedHeaders)
-                    .body(SuccessMessage) }
+                    .body(SuccessMessage)
+            }
             ?: ResponseEntity.status(404).body((Message("Failed during refreshing not found user")))
```

Comment:
Może pierdoła, ale wydaje mi się, że tekst bardziej powinien brzmieć "Failed during refreshing; user not found"

Reply:
Wydaje mi się, że to nawet ważne, żeby poprawić